bokehjs/src/coffee/tool/inspectors/inspect_tool.coffee

```diff
@@ -28,7 +28,8 @@ class InspectTool extends Tool.Model
   event_type: "move"
 
   nonserializable_attribute_names: () ->
-    super().concat(['event_type', 'inner_only'])
+    attrs = _.without(super(), 'active')
+    return attrs.concat(['event_type', 'inner_only'])
```

Comment:
> That (Override) seems better to me.

So I was only saying it could be viewed as an override. `Override` (property descriptor) doesn't belong here, because it's bokeh-side, whereas nonserializable properties are bokehjs-side. I'm not sure if we are on the same page and this code fine fine to proceed with?